docs/source/tutorial/interfaces.rst

```diff
@@ -645,7 +643,7 @@ Named Implementations
 =====================
 
 It can be desirable to have multiple implementations of an interface for the
-same type, for example to provide alternative methods for sorting or printing
+same type — for example to provide alternative methods for sorting or printing
```

Comment:
Again, shouldn't be an em-dash. The original is fine here imo.

Reply:
Reverted.